app/herms.hs

```diff
@@ -403,8 +412,13 @@ exportP  t = Export <$> severalRecipesP t <*> formatP t
 removeP  t = Remove <$> severalRecipesP t
 viewP    t = View   <$> severalRecipesP t <*> servingP t <*> stepP t <*> conversionP t
 shopP    t = Shop   <$> severalRecipesP t <*> servingP t
+findP    t = Find   <$> findRegxP t
 dataDirP _ = pure DataDir
 
+-- | @findRegxP returns a parser for a regex string
```

Comment:
I feel like `regx` is an uncommon abbreviation - perhaps `regex` would be more obvious? Also, this is missing the closing `@`. 
```suggestion
-- | @findRegexP@ returns a parser for a regex string
```
(If you accept this suggestion, please also change the function name just below)